automation_tools/__init__.py

```diff
@@ -850,6 +850,80 @@ def iso_install(
         return installer_options
 
 
+def iso_install_5(
+        admin_password=None, check_sigs=False, run_katello_installer=True):
+    """Installs Satellite 5 from an ISO image.
+
+    The following environment variables affect this command:
+
+    """
+    if isinstance(check_sigs, str):
+        check_sigs = (check_sigs.lower() == 'true')
+
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+
+    iso_url = os.environ.get('ISO_URL') or os.environ.get('BASE_URL')
+    if iso_url is None:
+        print('Please provide a valid URL for the ISO image.')
+        sys.exit(1)
+
+    # Download the ISO
+    print('Downloading ISO...')
+    iso_download(iso_url)
+    # Create a 'check-out' folder, mount ISO to it...
+    run('mkdir -p ISO')
+    run('mount *.iso ISO -t iso9660 -o loop')
+
+    # prepare the answer file
+    opts = {
+        'admin-email': os.environ.get('ADMIN_EMAIL') or '',
```

Comment:
Better to use `os.environ.get('ADMIN_EMAIL', '')`. This applies for every line in this dictionary.